color.cpp

```diff
@@ -56,6 +55,15 @@ color create_black_color()
   return color(0, 0, 0, 255);
 }
 
+color get_random_respawn_color(std::mt19937& ran_num_gen) noexcept
+{
+    int r = 1 + ran_num_gen()/((ran_num_gen.max() + 1u)/255);
+    int g = 1 + ran_num_gen()/((ran_num_gen.max() + 1u)/255);
+    int b = 1 + ran_num_gen()/((ran_num_gen.max() + 1u)/255);

```

Comment:
I'd be impressed if this actually works. Note that the range is *never* tested. I'll happily write an Issue to write those :-)